crates/hir-ty/src/mir.rs

```diff
@@ -267,13 +267,13 @@ impl ProjectionStore {
 }
 
 impl ProjectionId {
-    const EMPTY: ProjectionId = ProjectionId(0);
+    pub const EMPTY: ProjectionId = ProjectionId(0);
```

Comment:
not 100% sure about this, as it can be accessed indirectly via `From<LocalId>`..